dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops

```diff
@@ -222,6 +222,10 @@ MISSING_FEATURE_HAS_ALT_NAME=CWWKF0045E: An existing feature definition, {1}, is
 MISSING_FEATURE_HAS_ALT_NAME.explanation=The feature manager cannot find a definition for a feature specified in the server configuration. The name specified might match an existing feature definition.
 MISSING_FEATURE_HAS_ALT_NAME.useraction=Consider specifying the named, existing feature definition instead. 
 
+UPDATE_DISABLED_FEATURES_ON_CONFLICT=CWWKF0046W: The configuration includes an incompatible combination of features that is not supported. As a result the feature manager has not installed any features.
+UPDATE_DISABLED_FEATURES_ON_CONFLICT.explanation=The configuration includes an incompatible combination of features. To avoid invalid behavior the feature manager has disabled all features.
```

Comment:
Remove "has" so that the explanation has "feature manager disabled all features."